tests/ActiveRecordFactoryTest.php

```diff
@@ -26,13 +26,22 @@ public function testCreateAR(): void
 
     public function testCreateARWithConnection(): void
     {
-        $customerAR = $this->arFactory->createAR(Customer::class, $this->mysqlConnection);
+        $customerAR = $this->arFactory->createAR(arClass: Customer::class, db: $this->mysqlConnection);
         $db = $this->getInaccessibleProperty($customerAR, 'db', true);
 
         $this->assertInstanceOf(ConnectionPDOMysql::class, $db);
         $this->assertInstanceOf(Customer::class, $customerAR);
     }
 
+    public function testCreateARWithTableName(): void
+    {
+        $customerAR = $this->arFactory->createAR(Customer::class, 'customer');
+        $tableName = $customerAR->getTableName();
+
+        $this->assertSame('customer', $tableName);
+        $this->assertInstanceOf(Customer::class, $customerAR);
```

Comment:
```suggestion
        $customerAR = $this->arFactory->createAR(Customer::class, 'custom_table_name');
        $tableName = $customerAR->getTableName();

        $this->assertSame('custom_table_name', $tableName);
        $this->assertInstanceOf(Customer::class, $customerAR);
```

`customer` is default table name rely on class name